### DISCUSSION TITLE
Cannot rebind "Accept inline suggestion" in PyCharm

### DISCUSSION THREAD

**Discussion by itsthejoker** [community/community#6926] [Copilot Conversations] [answered]
Hi, all. Tab to accept a suggestion is massively screwing with my workflow and the documentation (https://github.com/github/copilot-docs/blob/main/docs/jetbrains/configuring.md) says that it should be configurable, but there is no option for it. 

![image](https://user-images.githubusercontent.com/5179553/139310940-ff7e040d-4c17-447b-8909-f481e44452ea.png)

How can I rebind this command to something less obtrusive?

**Comment by jansorg** [comment, marked as answer]
Thanks for reporting! Unfortunately, this isn't possible yet and the documentation isn't clear about this.
We'll track this internally to discuss this.

**Comment by itsthejoker** [comment]
I hope that it becomes configurable; I've disabled the plugin in the meantime because I'd rather not have it at all than have it intercept Tab when I don't want it to. Thanks for your help!

**Reply by jansorg** [reply]
Could you clarify how it's intercepting TAB when it's not wanted? 
For example, is it applying shown completions, but you wanted to indent the line instead?
Or would you prefer that TAB isn't triggering completions in the editor?
Thanks!

**Comment by itsthejoker** [comment]
I would prefer that TAB wasn't triggering completions at all, but yes,
specifically it's during indenting. I move code around a lot while
workshopping ideas, so it's really distracting when I want to move
something and instead get a whole new line of text.

Along the lines of accepting a completion, I would actually expect the
right arrow key to be the trigger (that's how the fish shell does it, for
instance -- but now that I try it, I see that TAB does also work there). If
that wasn't baked in though then I would remove the keybind altogether and
only use it through the alt+enter context menu.

On Mon, Nov 1, 2021 at 4:51 AM Joachim Ansorg ***@***.***>
wrote:

> Could you clarify how it's intercepting TAB when it's not wanted?
> For example, is it applying shown completions, but you wanted to indent
> the line instead?
> Or would you prefer that TAB isn't triggering completions in the editor?
> Thanks!
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/github/feedback/discussions/6926#discussioncomment-1568555>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABHQRILE7KJEEKLN4UZDNDTUJZIKJANCNFSM5G5P37NQ>
> .
>


-- 
"After silence, that which comes nearest to expressing the inexpressible is
music."

*Aldous Huxley*
